frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineTaskLogs.scss

```diff
@@ -3,6 +3,12 @@
     margin-top: var(--pf-global--spacer--lg);
   }
   &__container {
+    &.is-edge {
+      .pf-c-alert {
+        display: none;
+      }
+    }
```

Comment:
Not my finest code... but it does prevent this:

![EdgeLogs_Pipelines](https://user-images.githubusercontent.com/8126518/69831729-bdacc700-11f8-11ea-85cb-52838dd19fac.png)
![EdgeLogs_PipelinesExcessive](https://user-images.githubusercontent.com/8126518/69831730-bdacc700-11f8-11ea-99f5-89bb095bfa21.png)

Closing of the websocket causes Edge to throw an error which is caught by the error handling and shown on each `ResourceLog` that is mounted.